code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -60,23 +58,23 @@
 		return ..()
 
 
-/obj/structure/cult/proc/getETA()
+/obj/structure/cult/itemspawner/proc/getETA()
 	var/time = (cooldowntime - world.time)/600
 	var/eta = "[round(time, 1)] minutes"
 	if(time <= 1)
 		time = (cooldowntime - world.time)*0.1
 		eta = "[round(time, 1)] seconds"
 	return eta
 
-/obj/structure/cult/culttalisman
+/obj/structure/cult/itemspawner/talisman
 	name = "altar"
 	desc = "A bloodstained altar dedicated to a cult."
 	icon_state = "talismanaltar"
 	health = 150 //Sturdy
 	death_message = "<span class='warning'>The altar breaks into splinters, releasing a cascade of spirits into the air!</span>"
 	death_sound = 'sound/effects/altar_break.ogg'
 
-/obj/structure/cult/culttalisman/attack_hand(mob/living/user)
+/obj/structure/cult/itemspawner/talisman/attack_hand(mob/living/user)
```

Comment:
`attack_hand` can be all defined on the `/obj/structure/cult/itemspawner` level, removing a bunch of duplicated code